docker_wait_and_commit.sh

```diff
@@ -2,6 +2,7 @@
 #
 # Waits until a specific block (range) is reached, and then commits the Docker image.
 #
+SLEEP_TIME=60
```

Comment:
leave this at 30 plx

Reply:
@metachris bigger than 15 it will fail.
If you are expecting --until-block to be triggered you must iterate with less seconds than the current block generation.

It's just for the --until-block option. The rest keeps at 60 sec

Reply:
Ok, i get it... the default must be 30 and not 60.
I will make the desired changes.